### PR TITLE
feat(rollup): add silent attr to rollup_bundle to support --silent flag

### PR DESCRIPTION
**Pull request by josephperrott**
As rollup_bundle already supports specific rollup flags with specific
attributes on the rollup_bundle rule, adding a boolean attribute for
silent makes the most sense.

This boolean determines if the --silent flag is added to the rollup command
when called.

I was unable to add tests for this as its change is to the stdout of the execution
of bazel, not the results of the rollup command.

### CODE REVIEW

**Comment by alexeagle** [comment]
Note: related to #1661 which was also for `--silent`.

My take is: we wrap rollup in a custom rule, and attrs like `sourcemap` make sense for Bazel to know about since they affect the output files. So we should continue to mirror the Rollup API in attributes.

This has an unfortunate meaning in rollup - `--silent` also causes builds to pass that would have otherwise failed, because they treat many fatal conditions as a warning and you have to supply a custom handler to find out about these if you use `--silent` https://github.com/rollup/rollup/blob/master/docs/999-big-list-of-options.md#onwarn
so that's why we can't make the default here True, which would make it more Bazel-idiomatic.

File: packages/rollup/src/rollup_bundle.bzl [hunk 1]
```diff
@@ -168,6 +168,9 @@ Otherwise, the outputs are assumed to be a single file.
         cfg = "host",
         default = "@npm//rollup/bin:rollup",
     ),
+    "silent": attr.bool(
```
Comment (alexeagle) [inline packages/rollup/src/rollup_bundle.bzl]: Could you add a warning in this doc that using silent can cause rollup to ignore problems in your code, and link to https://github.com/rollup/rollup/blob/master/docs/999-big-list-of-options.md#onwarn and maybe https://github.com/rollup/rollup/issues/2686 ?

You could also mention that Bazel expects tools to print nothing on success, so enabling `--silent` gives a more Bazel-idiomatic experience, but the default here is False due to how it also drops warnings.

Comment (josephperrott) [reply packages/rollup/src/rollup_bundle.bzl]: Done, I added this information to the `attr` docstring.

File: packages/rollup/src/rollup_bundle.bzl [hunk 2]
```diff
@@ -168,6 +168,14 @@ Otherwise, the outputs are assumed to be a single file.
...
+        doc = """Whether to execute the rollup binary with the --silent flag, defaults to False.
+
+Using --silent can cause rollup to [ignore errors/warnings](https://github.com/rollup/rollup/blob/master/docs/999-big-list-of-options.md#onwarn) 
+which are only surfaced via logging.  Since bazel expects printing nother on success, setting silent to True
```
Comment (alexeagle) [inline packages/rollup/src/rollup_bundle.bzl]: nit: nother->nothing

**Pull request merged**